docs/api/textarea.md

```diff
@@ -29,6 +29,40 @@ import BasicPlayground from '@site/static/usage/v7/textarea/basic/index.md';
 
 <BasicPlayground />
 
+## Label Positioning
```

Comment:
Can we rename this "Label Placement"? I'd like to converge on a single naming (See https://github.com/ionic-team/ionic-docs/pull/2747)